src/styles/blocks/calendar-day.scss

```diff
@@ -0,0 +1,17 @@
+.calendar__day {
+  position: relative;
+  box-sizing: border-box;
```

Comment:
You have made `day` and element of `calendar` block. That's the right thing to do.
But in this case, styles should be in the same file.